_opensearch/rest-api/cluster-health.md

```diff
@@ -38,8 +38,8 @@ Parameter | Type | Description
 :--- | :--- | :---
 expand_wildcards | Enum | Expands wildcard expressions to concrete indices. Combine multiple values with commas. Supported values are `all`, `open`, `closed`, `hidden`, and `none`. Default is `open`.
```

Comment:
Can we change "indices" to "indexes" here?

Reply:
ok. 
as a followup q: should we keep the parameter 'indices' if this is coded this way?